### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'trim')

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#17546]
### Describe the bug

It happens after #17545 when user tries to delete pages.

### To Reproduce

1. Follow the steps from #17545
2. Click on 'Delete pages'
3. See error

### Expected behavior

It should not give any error.

### Screenshots

https://user-images.githubusercontent.com/25424343/169704810-7716a938-b8e1-400e-ad56-082a2e33471c.mp4

### Server configuration

- Operating system: Linux
- Web server: nginx/1.14.2
- Database version: 8.0.18 - MySQL Community Server - GPL
- PHP version: 7.4.29
- phpMyAdmin version: 5.3.0-dev

### Client configuration

- Browser: Chrome/101.0.4951.67
- Operating system: Windows 10

### Additional context

```JSON
{
    "pma_version": "5.3.0-dev",
    "browser_name": "CHROME",
    "browser_version": "101.0.4951.67",
    "user_os": "Win",
    "server_software": "nginx/1.14.2",
    "user_agent_string": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.67 Safari/537.36",
    "locale": "en",
    "configuration_storage": "disabled",
    "php_version": "7.4.29",
    "script_name": "index.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "Cannot read properties of undefined (reading 'trim')",
        "stack": [
            {
                "line": "342",
                "func": "",
                "context": [
                    "            if (lastException === ex) {",
                    "                processLastException();",
                    "            }",
                    "        }, (stack.incomplete ? 2000 : 0));",
                    "",
                    "        throw ex; // re-throw to propagate to the top level (and cause wind//...",
                    "    }",
                    "",
                    "    report.subscribe = subscribe;",
                    "    report.unsubscribe = unsubscribe;",
                    "    return report;"
                ],
                "uri": "js/vendor/tracekit.js?v=5.3.0-dev",
                "scriptname": "js/vendor/tracekit.js"
            },
            {
                "func": "HTMLButtonElement.<anonymous>",
                "line": "1011",
                "column": "77",
                "context": [
                    "      }",
                    "",
                    "      var modal = DesignerMove.displayModal(data.message, Messages.strSaveP//...",
                    "      $('#designerModalGoButton').on('click', function () {",
                    "        var $form = $('#save_as_pages');",
                    "        var selectedValue = $form.find('input[name=\"selected_value\"]').val(//...",
                    "        var $selectedPage = $form.find('select[name=\"selected_page\"]');",
                    "        var choice = $form.find('input[name=\"save_page\"]:checked').val();",
                    "        var name = '';",
                    "",
                    "        if (choice === 'same') {"
                ],
                "uri": "js/dist/designer/move.js?v=5.3.0-dev",
                "scriptname": "js/dist/designer/move.js"
            },
            {
                "func": "HTMLButtonElement.newFunc",
                "line": "258",
                "column": "23",
                "context": [
                    "   */",
                    "  wrapFunction: function (func) {",
                    "    if (!func.wrapped) {",
                    "      var newFunc = function () {",
                    "        try {",
                    "          return func.apply(this, arguments);",
                    "        } catch (x) {",
                    "          TraceKit.report(x);",
                    "        }",
                    "      };",
                    ""
                ],
                "uri": "js/dist/error_report.js?v=5.3.0-dev",
                "scriptname": "js/dist/error_report.js"
            },
            {
                "func": "HTMLButtonElement.dispatch",
                "line": "2",
                "column": "43064",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.3.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "HTMLButtonElement.v.handle",
                "line": "2",
                "column": "41048",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.3.0-dev",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            }
        ],
        "incomplete": "false",
        "partial": "true",
        "uri": "index.php?route=%2Ftable%2Fcreate"
    }
}
```

**Comment by williamdes** [comment]
Also when doing save as and the open page
Sending a fix for this right now

**Comment by liviuconcioiu** [comment]
@williamdes this issue should be reopened, because it can be reproduced in another way.

### To Reproduce

Steps to reproduce the behavior:

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/designer&db=sakila
2. Click on 'Save page as'
3. Click on 'Go'
2. Click on 'Save page'
3. Click on 'Go'
3. See error

### Screenshots

https://user-images.githubusercontent.com/25424343/216212597-6c4576dc-c6a0-4eca-9e21-96e98c3147c9.mp4

### Server configuration

- phpMyAdmin version: 5.2.1-dev, 5.3.0-dev

### Client configuration

- Browser: Chrome/112.0.5573.0
- Operating system: Windows 10

**Comment by williamdes** [comment]
It's best to open a new issue, because it causes for me a lot of management since the issue is already fixed and milestoned. It means that it needs urgent fixing to be able to close it before the release :)
